core/controller/src/main/scala/whisk/core/loadBalancer/LoadBalancerService.scala

```diff
@@ -294,7 +294,7 @@ class LoadBalancerService(
      * these are moved to some common place (like a subclass of Message?)
      */
     private def hashAndCountSubjectAction(msg: ActivationMessage): (Int, Int) = {
-        val subject = msg.subject().toString()
+        val subject = msg.user.subject()
```

Comment:
Gladly when the other pr is merged. It will normalize this to asString. FYI @psuter.